client/valor/schemas/symbolic/types.py

```diff
@@ -59,44 +60,35 @@ def __init__(
         name: str,
         key: typing.Optional[str] = None,
         attribute: typing.Optional[str] = None,
-        owner: typing.Optional[str] = None,
```

Comment:
docstrings for this class would be helpful. it's not obvious to me why we need dtype here and what its expected values are